pipeline/scripts/4/3/tier-1/test-rgw-single-site-upgrade.sh

```diff
@@ -5,7 +5,7 @@ random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
 instance_name="ci-${random_string}"
 platform="rhel-8"
 rhbuild="4.3"
-test_suite="suites/nautilus/rgw/rgw_test-update-4.2-to-4.x-latest.yaml"
+test_suite="suites/nautilus/rgw/tier1_rgw_test-4x-upgrade-to-latest.yaml"
```

Comment:
Could we rename the suite according to guideline? That is `tier-1_rgw_test-4x-upgrade-to-latest.yaml`... thanks for fixing it.

Reply:
I think this slipped during my review.